.circleci/config.yml

```diff
@@ -306,48 +290,32 @@ commands:
       - store_artifacts:
           path: /tmp/junit
 
-  build-atcp-assets:
-    description: build atcp auth0 scripts & styles
-    steps:
-      - run:
-          name: build atcp auth0 scripts
-          command: |
-            set -u
-            ng build ddp-atcp-auth
-            npm run convert-atcp-auth-build-to-UMD
-      - run:
-          name: build atcp auth0 styles
-          command: |
-            set -u
-            ./node_modules/.bin/sass ./projects/ddp-atcp/src/auth0/auth0-styles/main.scss ./projects/ddp-atcp/src/auth0/compiled/auth0-styles/main.css --style compressed
-
-  execute-study-specific-commands:
-    description: run study specific commands if necessary
-    parameters:
-      study_key:
-        type: string
-        default: "UNKNOWN"
-    steps:
-      - when:
-          condition:
-            equal: [ atcp, << parameters.study_key >> ]
-          steps:
-            - build-atcp-assets
-
-  build-app:
+  build:
```

Comment:
`build` is the new name for `build-app`. github diff viewer doesn't align changed code correctly.